locomotion-sdk/src/pages/AuthScreens/Onboarding/Name.js

```diff
@@ -13,6 +13,7 @@ const Name = () => {
     onboardingState, updateState, updateUserInfo, navigateBasedOnUser,
   } = onboardingContext.useContainer();
   const [showErrorText, setShowErrorText] = useState(false);
+  const [focusedField, setFocusedField] = useState('firstName');
```

Comment:
Why do we need it? we said the generic component should handle it

Reply:
accident, removed

Reply:
Cool